ipaserver/install/server/install.py

```diff
@@ -940,9 +940,22 @@ def install(installer):
 
     print("======================================="
           "=======================================")
-    print("Setup complete")
+    print("IPA server installation successful")
     print("")
-    print("Next steps:")
+    print("You can access Web UI using https://%s/ipa/ui" % (host_name))
+    print("\nObtain a Kerberos Ticket for issuing IPA command 'kinit admin'.")
+    print("\nServer is installed with following roles :")
+    print("* 389 Directory Server")
+    print("* Kerberos Server")
+    print("* HTTP Server")
```

Comment:
You have removed CA and DNS completely. That's not what I meant. You should print a line for CA and DNS, when the user has selected CA or DNS in the installer.